src/js/modules/articles/components/ArticleFooter.js

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import injectSheet from "react-jss";
+import { Row, Col } from "react-bootstrap/lib";
+import { toRoman } from "roman-numerals";
+
+import SubscriptionForm from "../../accounts/components/forms/SubscriptionForm";
+import { subscribe } from "../../accounts/actions";
+
+const styles = {
+  ArticleFooter: {
+    fontFamily: "Minion Pro",
+  },
+  inPrint: {
+    borderBottom: "1px solid #ddd",
+    color: "#000",
+    fontSize: "19px",
+    fontStyle: "italic",
+    lineHeight: 1.4,
+    marginBottom: "14px",
+    paddingBottom: "13px",
+    "& a": {
+      color: "#000",
+      textDecoration: "underline",
+      "&:hover, &:active, &:focus": {
+        color: "#000",
+      },
+    },
+  },
+  subscriptionCTA: {
+    color: "#000",
+    fontSize: "19px",
+    marginBottom: "10px",
+  },
+  subscriptionFormContainer: {
+    marginBottom: "21px",
+    "& > div > form": {
+      display: "flex",
+    },
+    "& > div > form > div": {
+      display: "inline-block",
+    },
+    "& > div > form > div:first-child": {
+      width: "80%",
+      "& > div > input": {
+        // email input
+        borderRadius: "3px 0 0 3px",
+        width: "100%",
+      },
+    },
+    "& > div > form > div:last-child": {
+      width: "20%",
+      "& > button": {
+        // "Subscribe" button
+        borderRadius: "0 3px 3px 0",
+        margin: 0,
+        width: "100%",
+      },
+    },
+  },
+  "@media (max-width: 991px)": {
+    ArticleFooter: {
+      padding: "0 10%",
+    },
+  },
+  "@media (max-width: 767px)": {
+    ArticleFooter: {
+      padding: "0 2%",
+    },
+  },
+};
+
+const ArticleFooter = ({ classes, article, subscribe }) => {
+  return (
+    <Row className={classes.ArticleFooter}>
+      <Col xs={12} sm={12} md={8} lg={8}>
+        <p className={classes.inPrint}>
+          This article appears in print in&nbsp;
+          <a
+            className={classes.subscribe}
+            href="https://issuu.com/stuyspectator/docs"
+            target="_blank"
+          >
+            {`Volume ${toRoman(article.volume)}, Issue ${article.issue}`}
+            {/* TODO: Lookup table for individual volume/issue links */}
+          </a>
+          .
+        </p>
+        <p className={classes.subscriptionCTA}>
+          Never miss a <i>Spectator</i> issue again. Sign up for for our
+          newsletter to receive a weekly e-mail with staff-picked stories.
+        </p>
+        <div className={classes.subscriptionFormContainer}>
+          <SubscriptionForm
+            onSubmit={values => subscribe(values)}
+            callToAction="Go"
```

Comment:
callToAction? What is this? A 19th century evangelical text?

Reply:
It's a common term in UI/UX design. A call to action is the button that says "Subscribe!", "See movie!", etc. In this case, the `callToAction` prop is the text that goes in the submit button. The length of the word "Subscribe" (which is the default `SubscriptionForm` call to action) makes the CSS of the button in the `ArticleFooter` fairly difficult. As a result, I gave the `SubscriptionForm` an option for customizability in its call to action.

Reply:
Eh, using UI/UX terms is a little tricky. I'd prefer a more straightforward term that doesn't require a glossary or google search to understand. Consider `label` or `submitLabel`